src/test/fuzz.cpp

```diff
@@ -43,26 +45,25 @@
 namespace stellar
 {
 
-struct CfgDirGuard
+Config
+confGen(int instanceNumber)
```

Comment:
why is this function in the global namespace? It's only used below, right?

Reply:
Done -- use a lambda instead now.